resources/crds/ripsaw_v1alpha1_fio_distributed_vm_cr.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: fio-vm-benchmark
+  namespace: my-ripsaw
+spec:
+#  elasticsearch:
+#    server: <es_server>
+#    port: 9200
+#  clustername: myk8scluster
+#  test_user: ripsaw
+  workload:
+    name: "fio_distributed"
+    args:
+      # if true, do large sequential write to preallocate volume before using
+      prefill: true
+      # number of times each test
+      samples: 3
+      # number of fio pods generating workload
+      servers: 3
+      # put all fio pods on this server
+      pin_server: ''
+      # Run fio servers in VMs
+      kind: vm
+      # Specify kubevirt registry image for server vm
+      # image_vm:
```

Comment:
Specifying diff image and image location did not work using image_vm:. The default image download was tried had to edit pod to get desired image to be used.

Reply:
yes you're right, I called it `vm_image` in the code... will fix the CR